web-console/knikubevirt/guest-agent-not-installed/guest-agent-not-installed.md

```diff
@@ -0,0 +1,22 @@
+#Guest Agent not installed
+
+The purpose of this document is to show what a VM would look like without the guest agent installed. 
+
+In the list view we show an “info” icon with the status with a tooltip that show that the guest agent isn’t installed. Further information can be provided in a tooltip.
```

Comment:
@andybraren that's a great idea! I don't recall exactly but when this design was done I don't think we had a standard. Are we saying we won't have tooltips at all for list view? the one thing that distinguishes them is if they contain really important info or just helpful info. In this case if we provide all the relevant info in the status popup dialog maybe we can either stick with a tooltip for the IP or remove it entirely.

Reply:
> Thanks Yifat!
> 
> Since this is to closely related, I wonder if this documentation should go into the "States and Actions" section of the main [VM List doc](http://openshift.github.io/openshift-origin-design/web-console/knikubevirt/vm-list/vm-list) under a new "Guest agent not installed" heading instead of being a separate page.

Hi @andybraren, np. it does make sense to me as well they should go together. The only question I have is that I think the VM List PR had already been merged into Openshift, so how do I do that? Thanks.

Reply:
@yfrimanm you should be able to just remove this file entirely and just add it's content to the vm-list.md file.

Reply:
@andybraren @yfrimanm @lizsurette after looking at the design for the list view I think we should hold off on merging this into that for now. Why don't we get this updated with the new status dialog (which is different from what we have in the list view designs currently) and merge. I will create an issue to go pull this design into the list view design once we update those designs. Make sense?

Reply:
> @andybraren @yfrimanm @lizsurette after looking at the design for the list view I think we should hold off on merging this into that for now. Why don't we get this updated with the new status dialog (which is different from what we have in the list view designs currently) and merge. I will create an issue to go pull this design into the list view design once we update those designs. Make sense?

@matthewcarleton That makes total sense to me!